frontend/src/App.tsx

```diff
@@ -2,19 +2,22 @@ import {Route, Routes} from "react-router-dom";
 import TeacherHomePage from "./teacherPage/TeacherHomePage";
 import StudentList from "./teacherPage/StudentList.tsx";
 import HomeworkAdd from "./teacherPage/HomeworkAdd.tsx";
+import StudentHomePage from "./studentPage/StudentHomePage.tsx";
 
 export default function App() {
+    const studentId: string="1";
 
-
-  return (
+    return (
     <>
-        <TeacherHomePage/>
-        <h1>App</h1>
+        {/* <TeacherHomePage/> */}
```

Comment:
do you need the comments?

Reply:
I need to combine it with the login page.